stratum/lib/constants.h

```diff
@@ -34,8 +33,10 @@ constexpr char kDefaultAuthPolicyFilePath[] =
 constexpr char kLocalStratumUrl[] = "localhost:28000";
 
 // This URL is used by external gNMI, gNOI and P4Runtime clients.
-// This TCP port 9339 is an IANA-reserve port for gNMI and gNOI.
-constexpr char kExternalStratumUrls[] = "0.0.0.0:28000,0.0.0.0:9339";
+// This TCP port 9339 is an IANA-reserved port for gNMI and gNOI.
```

Comment:
Super minor nit...  Maybe also a remark to port 28000 can be added also explaining that it is deprecated and will be removed in the near future?  Also maybe the 'This' from 'This TCP port 9339' can be removed to be consistent with the line below?

Reply:
We'll do the deprec notice in the release notes too.